packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -73,18 +73,30 @@ spec:
       coreDNSPluginImage: {{ template "system_default_registry" . }}{{ .Values.istiocoredns.pluginImage.repository }}:{{ .Values.istiocoredns.pluginImage.tag }}
       coreDNSTag: {{ .Values.istiocoredns.image.tag }}
     mixer:
-      {{- if .Values.policy.enabled }}
+      {{- if .Values.mixer.policy.enabled }}
       policy:
-        image: {{ template "system_default_registry" . }}{{ .Values.policy.repository}}:{{ .Values.policy.tag }}
+        image: {{ template "system_default_registry" . }}{{ .Values.mixer.policy.repository }}:{{ .Values.mixer.policy.tag }}
       {{- end }}
-      {{- if .Values.telemetry.enabled }}
+      {{- if .Values.mixer.telemetry.enabled }}
       telemetry:
-        image: {{ template "system_default_registry" . }}{{ .Values.telemetry.repository}}:{{ .Values.telemetry.tag }}
+        image: {{ template "system_default_registry" . }}{{ .Values.mixer.telemetry.repository }}:{{ .Values.mixer.telemetry.tag }}
       {{- end }}
     {{- if .Values.pilot.enabled }}
     pilot:
-      image: {{ template "system_default_registry" . }}{{ .Values.pilot.repository}}:{{ .Values.pilot.tag }}
+      image: {{ template "system_default_registry" . }}{{ .Values.pilot.repository }}:{{ .Values.pilot.tag }}
     {{- end }}
+    {{- if or .Values.telemetry.v1.enabled .Values.telemetry.v2.enabled }}
```

Comment:
Is this line necessary ? Even if they are both false they would just both be false below.